build.gradle.kts

```diff
@@ -64,29 +63,19 @@ kotlin {
                 implementation("org.junit.jupiter:junit-jupiter-engine:5.7.0")
             }
         }
-        val jvmTest by getting {
-            dependencies {
-                implementation("org.junit.jupiter:junit-jupiter:5.7.0")
-                implementation("org.junit.jupiter:junit-jupiter-api:5.7.0")
-                implementation("org.junit.jupiter:junit-jupiter-engine:5.7.0")
-            }
-        }
-        val androidTest by getting {
-            dependencies {
-                implementation("org.junit.jupiter:junit-jupiter:5.7.0")
-                implementation("org.junit.jupiter:junit-jupiter-api:5.7.0")
-                implementation("org.junit.jupiter:junit-jupiter-engine:5.7.0")
-            }
-        }
+        val jvmMain by getting
+        val jvmTest by getting
+        val androidMain by getting
+        val androidTest by getting
     }
 }
 
 android {
-    compileSdkVersion(29)
+    compileSdk = 31
     sourceSets["main"].manifest.srcFile("src/androidMain/AndroidManifest.xml")
     defaultConfig {
-        minSdkVersion(21)
-        targetSdkVersion(29)
+        minSdk = 21
+        targetSdk = 31
```

Comment:
Hey, sorry for taking this long to answer, either GitHub didn't notify me or I missed the notification :(

Anyway, thanks for the contribution! My only question is about the `targetSdk`. Any reason for bumping it from 29 to 31? It's been a looooong time since I developed for Android so I don't remember what it'll affect in the end. If it doesn't affect much, I don't see why not.

Reply:
31 is for android 12
32 is for android 12L

Its best to target the latest releases at all times, and always maintain the latest min version.